public/css/forms.css

```diff
@@ -181,33 +181,48 @@ input[type="radio"] {
 
 .checkmark {
   background-color: rgba(255, 255, 255, 1);
-  height: 32px;
-  width: 32px;
-  border-radius: 4px;
-  display: block;
+  height: 24px;
+  width: 24px;
+  border-radius: 100%;
```

Comment:
This may result in less ideal rounded corners on longer text strings. Would it possible to use more specific (in `px`) rounded corner radius?